scaling_performance/optimizing_storage.adoc

```diff
@@ -11,19 +11,167 @@
 
 toc::[]
 
-== Optimizing Storage
+This topic provides information for optimizing OpenShift storage.
+
+[[general-storage-guidelines]]
+== General Storage Guidelines
+
+The following table lists the persistent storage back-end types {product-title} can be used with.
+
+.Available storage options
+[options="header"]
+|===
+| Storage type | Description | Examples
+
+| Block
+| This type of storage is presented to the operating system as a block device. It is
+suitable for applications that need full control of storage and operate at a low level on files
+bypassing the file system. It is sometimes also referred to as Storage Area Network (SAN). It is
+usually a non-shareable type of storage, which means that only one client at a time can
+mount an endpoint of this type.
+| CNS/CRS GlusterFS footnoteref:[disclaimer, CNS/CRS GlusterFS, Ceph RBD, OpenStack Cinder, AWS EBS, Azure Disk, GCE persistent disk, and VMware vSphere support dynamic PV provisioning natively in OpenShift Container Platform.] , iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS footnoteref:[disclaimer], Dell/EMC Scale.IO,
+VMware vSphere Volume, GCE Persistent Disk footnoteref:[disclaimer], Azure Disk
+
+| File
+| This type of storage is presented to the operating system as a file system export to be mounted. File storage is
+sometimes also referred to as Network Attached Storage (NAS). Concurrency, latency, file locking mechanisms and other
+capabilities will vary widely between protocols, implementations, vendors, and scales.
```

Comment:
maybe s/capabilities will vary widely/capabilities varies widely

Reply:
If the latter then it's 'capabilities vary widely'